src/libPersistence/BlockStorage.cpp

```diff
@@ -135,6 +135,11 @@ bool BlockStorage::PutTxBody(const dev::h256& key,
     LOG_MARKER();
 
 #ifndef IS_LOOKUP_NODE
+    if (!m_txBodyDBs.size())
```

Comment:
use m_txBodyDBs.empty() or !m_txBodyDBs.size() == 0

Use ! for boolean  and == for numerical comparsion